.ci/pipeline/build-and-test-lnx.yml

```diff
@@ -27,47 +28,73 @@ steps:
       conda config --add channels conda-forge
       conda config --set channel_priority strict
       conda update -y -q conda
-      conda create -q -y -n CB -c conda-forge python=$(PYTHON_VERSION) dal-devel mpich pyyaml "dpcpp-cpp-rt=2024.2.0"
+      conda create -q -y -n CB -c conda-forge python=$(PYTHON_VERSION) mpich pyyaml "dpcpp-cpp-rt=2024.2.0"

```

Comment:
Question: would this switch _all_ of these jobs to the external builds? I think it'd be quite useful to leave some with the stable release too.

Reply:
Very good question and suggestion. Why do we stop using stable version for building and testing?

Reply:
It will switch to external builds only if a DALROOT is specified, and example using it is here: https://github.com/oneapi-src/oneDAL/pull/2950 which is the immediate use case.  The question remains the same as when GitHub actions CI was introduced, do we eventually want to move toward using versions of oneDAL main branch rather than release branches?

Reply:
@icfaust I think it would be good to get @napetrov and @syakov-intel ’s opinions on this.

Reply:
Thank you @icfaust for the clarification. For me it is resolved.